README.md

```diff
@@ -51,6 +51,16 @@ description: <(optional) Text appearing under the section name in the section li
 ---
 ```
 
+To add alert box just put in your markdown :
+``` html
+  <div class="alert alert-infos">
```

Comment:
Nitpicking. "infos" (with an S for the plural) is frenchie-french. In English, the plural for info is info.